kafka_consumer/tests/conftest.py

```diff
@@ -106,33 +145,40 @@ def kafka_instance():
 def create_topics():
     client = _create_admin_client()
 
+    if set(TOPICS).issubset(set(client.list_topics(timeout=1).topics.keys())):
+        return True
+
     for topic in TOPICS:
-        client.create_topics([NewTopic(topic, 2, 1)], operation_timeout=1, request_timeout=1)
+        client.create_topics([NewTopic(topic, 2, 1)])
+        time.sleep(1)
+
+    return set(TOPICS).issubset(set(client.list_topics(timeout=1).topics.keys()))
```

Comment:
Small nit: could you add a comment explaining what this logic is doing?

Reply:
Sure!